Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -99,7 +99,7 @@ private void ProcessKeyBindings()
             }
         }
 
-        private void KeywordRecognizer_OnPhraseRecognized(PhraseRecognizedEventArgs args)
+        private void KeywordRecognizer_OnPhraseRecognized(UnityEngine.Windows.Speech.PhraseRecognizedEventArgs args)
```

Comment:
This was required because of name clashing. Renaming new classes following discussion at https://github.com/Microsoft/HoloToolkit-Unity/pull/354#discussion_r90693128